### DISCUSSION TITLE
Publish SHA sums files among the build artifacts?

### DISCUSSION THREAD

**Discussion by sid-the-sloth** [fastfetch-cli/fastfetch#753] [General]
Is it possible to publish the shasum files among the build artifacts? Thanks!

**Comment by CarterLi** [comment]
Yes it is possible, but it doubles the number of artifacts. why do you want that?

**Reply by sid-the-sloth** [reply]
You can publish one file for SHA256 sums and/or one file for SHA512 sums, so it's _only_ 2 files max.

Example (see SHA256SUMS file) here: https://github.com/JetpackDuba/Gitnuro/releases/tag/v1.3.1

**Reply by CarterLi** [reply]
What's the benefit?

**Reply by sid-the-sloth** [reply]
The benefit is that in my installer script, I can simply download the SHASUM file to check the integrity of the downloaded package. Otherwise I have to scrape the html page and extract the checksum needed. Many projects here on github publish one or two checksum files... it just makes sense from a programmer's perspective.

Also, the creation of these files can be automated during the build, so it's only a one-time change, really.
